### ISSUE TITLE
README needs more details on Features

### ISSUE THREAD

**Issue by NukeManDan** [ipfs-shipyard/ipfs-desktop#1670]
**Describe the bug**
https://github.com/ipfs-shipyard/ipfs-desktop#features

On the readme, I am so excited by the fatures laid out to start/stop a node, and automatically get the ipfs CLI and keep it updated 🤩 ... but I don't see anything in the GUI that reflects these features... It would be very good to expose this more in the readme and docs (https://docs.ipfs.io/install/ipfs-desktop/) 🙏🏼

**Comment by NukeManDan** [comment]
Note: I am on linux, so this may be partly the issue?

**Comment by jessicaschilling** [comment]
Good point that the Linux tray issue could benefit from being explicitly noted. Thank you!

**Comment by jessicaschilling** [comment]
https://github.com/ipfs-shipyard/ipfs-desktop/pull/1703 should ameliorate a lot of this.

Note: Once that PR is approved/merged, I'll add some highlights (primarily features list) to https://docs.ipfs.io/install/ipfs-desktop/ in a separate PR in the docs repo.

**Comment by jessicaschilling** [comment]
NB: The PR for adding features summary to docs is at https://github.com/ipfs/ipfs-docs/pull/525.
